ArduCopter/APM_Config.h

```diff
@@ -35,6 +35,7 @@
 //#define MODE_RTL_ENABLED DISABLED                 // disable rtl mode support
 //#define MODE_SMARTRTL_ENABLED DISABLED            // disable smartrtl mode support
 //#define MODE_SPORT_ENABLED DISABLED               // disable sport mode support
+//#define DEVO_TELEM_ENABLED DISABLED
```

Comment:
This exact change should also be done in both copter and rover.

Reply:
Those files are empty so the best choice to not touch them

Reply:
What do you think about this @OXINARF  ?

Reply:
I agree with @night-ghost, those files are empty so no need to add it there. If someone wants to do a PR to fill them up, that's fine, but it's not a request for this PR.